app/helpers/receiver_helper.rb

```diff
@@ -1,11 +1,6 @@
 module ReceiverHelper
+  # Convert DOS, *NIX and MacOs
```

Comment:
If we need a comment, is the method properly named?

Reply:
YEp, you're right.